src/components/SettingsDialog/Components/AvatarSettings.tsx

```diff
@@ -97,12 +97,11 @@ export const AvatarSettings: FC<AvatarSettingsProps> = ({id}) => {
       <div className="avatar-settings__settings-wrapper">
         <div className="avatar-settings__settings">
           {Object.entries(settingGroups).map(([label, props], groupIndex, array) => (
-            <>
+            <div key={label}>
```

Comment:
You can also use `<Fragment key={label}>` instead of the tag. This will omit the visible `div` tag and the error will not appear.
https://reactjs.org/docs/fragments.html#keyed-fragments
Additionally it might be possible to create an HR element with CSS only and place it as ::before/::after element of :nth-child() **_OR_** maybe it's possible to use the CSS `gap` attribute and style it to let it look like an HR.

TLDR:
Solution is fine, but `<Fragment key={label}>` might be better, and I think the hardcoded `<hr>` in line 135/133 should be replaced by pure CSS. Then neither (Fragment/div) is required.